Codeinwp/neve
# fix: code tag style inconsistencies and dark mode readability [#2286]

### Summary

Fixes `code` tag style inconsistencies.

### Will affect visual aspect of the product

YES

### Screenshots 
![image](https://user-images.githubusercontent.com/15010186/106443709-7a498a80-6485-11eb-858f-e4b0ecc98b8e.png)


### Test instructions


- Create a new page, add blocks (table, paragraph, lists, etc.) and insert text inside them; 
- Apply the Inline Code style from the editor dropdown after selecting parts of the text;
- Code style should be consistent across all blocks;
- The code block itself has a bit of a different style than the inline code tags;
- The code tags should work fine and be readable using both the Base color palette and the Dark mode color palette;


Closes #2286.
